lib/internal/Magento/Framework/Reflection/Test/Unit/Fixture/TSample.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\Reflection\Test\Unit\Fixture;
+
+class TSample implements TSampleInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public function getPropertyName()
+    {
+        return '';
+    }
+
+    /**
+     * @inheritdoc
```

Comment:
Would be really nice to add test case with `{@inheritdoc}` to make it explicit such syntax is not supposed to work. I do not insist though, this PR is already excellent.

Reply:
I thought about it, but a lot of Magento code has such annotation. The current unit test covers such case for parsing return tag because the behavior just checks if a method has `@return` annotation.

Reply:
> a lot of Magento code has such annotation

We should fix it anyway. Not saying it must be a scope of this PR, but it could be.

Reply:
Absolutely agree, even more, I think should be a static test which won't allow adding this annotation.